test/helper/restart.js

```diff
@@ -0,0 +1,27 @@
+module.exports = function (rq) {
```

Comment:
We should do the exact same thing that we are already doing in the front end https://github.com/strapi/strapi/blob/master/packages/strapi-helper-plugin/lib/src/utils/request.js#L79